vendors/st/boards/stm32l475_discovery/ports/pkcs11/iot_pkcs11_pal.c

```diff
@@ -323,12 +323,14 @@ CK_OBJECT_HANDLE PKCS11_PAL_SaveObject( CK_ATTRIBUTE_PTR pxLabel,
 {
     CK_OBJECT_HANDLE xHandle = eInvalidHandle;
     CK_RV xBytesWritten = 0;
-    CK_RV xReturn = CKR_OK;
     uint32_t ulFlashMark = ( pkcs11OBJECT_PRESENT_MAGIC | ( ulDataSize ) );
-    uint8_t * pemBuffer = NULL;
-    size_t pemLength = 0;
-    CK_BBOOL xIsDestroy = CK_TRUE;
-    int temp;
+    #ifdef USE_OFFLOAD_SSL
+        CK_RV xReturn = CKR_OK;
+        uint8_t * pemBuffer = NULL;
+        size_t pemLength = 0;
+        CK_BBOOL xIsDestroy = CK_TRUE;
+        int temp;
+    #endif
```

Comment:
Can this `#endif` followed immediately by another `#ifdef USE_OFFLOAD_SSL` be combined, or are they intentionally separate to separate declarations and code?

Reply:
Yes, kept them separate to keep declarations and code separate.